core/org.wso2.carbon.tomcat.ext/src/main/java/org/wso2/carbon/tomcat/ext/transport/CarbonResponseWrapper.java

```diff
@@ -98,7 +98,11 @@ public String generateCookieString(Cookie cookie) {
 
         String cookieString = super.generateCookieString(cookie);
         if (cookie instanceof ServletCookie) {
-            cookieString = cookieString + "; SameSite=" + ((ServletCookie) cookie).getSameSite().getName();
+            if (((ServletCookie) cookie).getSameSite() == null) {
+                cookieString = cookieString + "; SameSite=" + SameSiteCookie.STRICT.getName();
```

Comment:
If SameSite attribute is not defined for the cookie, default value for SameSite attribute is `LAX`. Any specific reason why we're setting it to `STRICT`?

Reply:
It was added to maintain consistency with https://github.com/wso2/carbon-kernel/pull/2921/files#diff-77713c4d070323936e7d67d119fc629d4e3f7324554de48de7adb13821c0c79cR107